### PR TITLE
Adding the tag service and the audit table

### PR DESCRIPTION
**Pull request by vans13**
During this pull, I have done all the tag service, so that a moderator and an administrator can perform CRUD functions for the tags, also it allows you to get all tags sorted or just one by id.
On the other hand, I created the audit table already checked from the local machine, allowing to use triggers and SP.
Finally, an error on the url side is updated.

### CODE REVIEW

File: dtos/tagsDtos/tagsDto.go
Comment (santorar) [inline dtos/tagsDtos/tagsDto.go:1]: you can just return the tagDto in a array

Comment (vans13) [reply dtos/tagsDtos/tagsDto.go:1]: ok, I'm doing it

File: services/userServices.go
Comment (santorar) [inline services/userServices.go:1]: remember to get the back url from the .env

**Review by santorar — APPROVED** [review]
thanks

**Pull request merged**